src/main/java/frc/trigon/robot/constants/CameraConstants.java

```diff
@@ -1,11 +1,69 @@
+import frc.trigon.robot.poseestimation.apriltagcamera.AprilTagCamera;
+import frc.trigon.robot.poseestimation.apriltagcamera.AprilTagCameraConstants;
+import org.photonvision.simulation.SimCameraProperties;
 
 public class CameraConstants {
     public static final double
-            TRANSLATIONS_STD_EXPONENT = 0.005,
-            THETA_STD_EXPONENT = 0.01;
+            TRANSLATIONS_STD_EXPONENT = 0.02,
+            THETA_STD_EXPONENT = 0.02;
 
-    public static final ObjectDetectionCamera NOTE_DETECTION_CAMERA = new ObjectDetectionCamera("Collection Camera");
+    private static final Transform3d
+            FRONT_CENTER_TO_CAMERA = new Transform3d(
+            new Translation3d(0.325, 0.0465, 0.192),
+            new Rotation3d(0, Units.degreesToRadians(-31), 0)
+    ),
+            REAR_CENTER_TO_CAMERA = new Transform3d(
+                    new Translation3d(-0.325 + 0.00975, 0, 0.095),
+                    new Rotation3d(Math.PI - Units.degreesToRadians(0), Units.degreesToRadians(-33), Math.PI + Units.degreesToRadians(0))
+            );
+
+    private static final int
+            CAMERA_WIDTH = 1600,
+            CAMERA_HEIGHT = 1200,
+            CAMERA_FPS = 60,
+            AVERAGE_CAMERA_LATENCY_MILLISECONDS = 35,
+            CAMERA_LATENCY_STANDARD_DEVIATIONS_MILLISECONDS = 5;
+    private static final Rotation2d CAMERA_FOV = Rotation2d.fromDegrees(90);
+    private static final double
+            AVERAGE_PIXEL_ERROR = 0.25,
+            PIXEL_STANDARD_DEVIATIONS = 0.08;
+    public static final boolean SHOULD_USE_CAMERA_SIMULATION = true;
+    public static final SimCameraProperties SIM_CAMERA_PROPERTIES = new SimCameraProperties();
+
+    static {
+        configureSimCameraProperties();
+    }
+
+    public static final AprilTagCamera
+            FRONT_TAG_CAMERA = new AprilTagCamera(
+            AprilTagCameraConstants.AprilTagCameraType.PHOTON_CAMERA,
+            "FrontTagCamera",
+            FRONT_CENTER_TO_CAMERA,
+            THETA_STD_EXPONENT,
+            TRANSLATIONS_STD_EXPONENT
+    ),
+            REAR_TAG_CAMERA = new AprilTagCamera(
+                    AprilTagCameraConstants.AprilTagCameraType.PHOTON_CAMERA,
+                    "RearTagCamera",
+                    REAR_CENTER_TO_CAMERA,
+                    THETA_STD_EXPONENT,
+                    TRANSLATIONS_STD_EXPONENT
+            );
+    public static final ObjectDetectionCamera NOTE_DETECTION_CAMERA = new ObjectDetectionCamera("NoteDetectionCamera");
+
+    private static void configureSimCameraProperties() {
+        SIM_CAMERA_PROPERTIES.setCalibration(CAMERA_WIDTH, CAMERA_HEIGHT, CAMERA_FOV);
+        SIM_CAMERA_PROPERTIES.setCalibError(AVERAGE_PIXEL_ERROR, PIXEL_STANDARD_DEVIATIONS);
+        SIM_CAMERA_PROPERTIES.setFPS(CAMERA_FPS);
+        SIM_CAMERA_PROPERTIES.setAvgLatencyMs(AVERAGE_CAMERA_LATENCY_MILLISECONDS);
+        SIM_CAMERA_PROPERTIES.setLatencyStdDevMs(CAMERA_LATENCY_STANDARD_DEVIATIONS_MILLISECONDS);
```

Comment:
are you sure you don't need to call 
`SIM_CAMERA_PROPERTIES.setExposureTimeMs();` and
`SIM_CAMERA_PROPERTIES.setCalibration();` ?

Reply:
It's not used for the Object detection camera

Reply:
What about the object detection camera?